packages/lib/server-only/user/delete-user.ts

```diff
@@ -13,10 +13,33 @@ export const deleteUser = async ({ email }: DeleteUserOptions) => {
     },
   });
 
+  const defaultDeleteUser = await prisma.user.findFirst({
```

Comment:
probably make this `deletedServiceAccount` or similar. We could probably also store this as a method in our lib or something.

So then we would do something like `const serviceAccount = await deletedServiceAccount();` 

Could be nice 🤔